mix.exs

```diff
@@ -34,7 +34,8 @@ defmodule Saxy.MixProject do
   defp deps() do
     [
       {:ex_doc, "~> 0.16", only: :dev, runtime: false},
-      {:stream_data, "~> 0.5", only: [:dev, :test]}
+      {:stream_data, "~> 0.5", only: [:dev, :test]},
+      {:dialyxir, "~> 1.0", only: [:dev], runtime: false}
```

Comment:
Thanks for the fix! It looks perfect.

Only small nitpick: I'd prefer not to have `dialyxir` as a dependency at the moment.

Reply:
done! :-)